### PR TITLE
Move issue template to new GitHub system

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

Moved to the new templates system defined by GitHub to allow more than 1 template.

**To test:**

Code review. Check the wording of the new bug report template.

*This does not require release notes* because **it is an internal process change.** 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (DanielMurphy22) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: Is the idea with all of these descriptions that the person creating the issue is slowed down to think of a good way to explain it? If we don't want to force people to delete all these descriptions, then we could use `<!--` to set these as hidden comments, which might be nice here!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable/possible, add screenshots to help explain your problem.
```
Comment (DanielMurphy22) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: I assume this is to inspire some creativity! You could also mention adding a GIF of the issue (like from ScreenToGIF) if appropriate!

Comment (DanielMurphy22) [reply .github/ISSUE_TEMPLATE/bug_report.md:24]: e.g. https://github.com/mantidproject/mantid/issues/28306#issuecomment-696756489

**Review by DanielMurphy22 — CHANGES_REQUESTED** [review]
I added this to my repo to see how it works (I don't know if you can create issues [here](https://github.com/DanielMurphy22/SmokeTests/issues)).

Overall it's really good to have a change and happy to discuss the different comments I've made!

**Comment by DavidFair** [comment]
I'm going to clear the release milestone, as this can go in independent of a release. Feel free to give me a shout if you disagree

**Comment by martyngigg** [comment]
These can be seen live by clicking new at https://github.com/martyngigg/sandbox/issues.

**Review by DanielMurphy22 — APPROVED** [review]
Looks really good to me! Particularly like the link to the forum!

**Comment by DavidFair** [comment]
I like the fact we can automatically categorise things too, i.e. bugs :+1:

**Pull request merged**